docs/source/api/v1/cdns_name_snapshot.rst

```diff
@@ -321,6 +321,11 @@ Response Structure
 	:date:       The UNIX epoch timestamp date in the Traffic Ops server's own timezone
 	:tm_host:    The :abbr:`FQDN (Fully Qualified Domain Name)` of the Traffic Ops server
 	:tm_path:    A path relative to the root of the Traffic Ops server where a request may be replaced to have this :term:`Snapshot` overwritten by the current *configured state* of the CDN
+
+		.. deprecated:: ATCv5.1
+
+			This information should never be used; instead all tools and (especially) components **must** use the documented API. This field was removed in APIv4
+
```

Comment:
is there also a `cdns_name_snapshot_new.rst` for v1 that needs updating like the other versions as well?

Reply:
yeah, that should've been done.